src/sss_client/common.c

```diff
@@ -1145,7 +1193,7 @@ const char *ssscli_err2string(int err)
             return _("Unexpected format of the server credential message.");
             break;
         case ESSS_SERVER_NOT_TRUSTED:
-            return _("SSSD is not run by root.");
+            return _("SSSD is not trusted.");
```

Comment:
Hi,

maybe it would be more explicit to say `SSSD is not run by trusted user`?

bye,
Sumit

Reply:
Sure thing, fixed.

Reply:
Thanks